src/test/java/uk/gov/hmcts/reform/civil/service/DeadlinesCalculatorTest.java

```diff
@@ -323,4 +324,12 @@ private BankHolidays loadFixture() throws IOException {
         String input = ResourceReader.readString("/bank-holidays.json");
         return new ObjectMapper().readValue(input, BankHolidays.class);
     }
+
+    @Test
+    public void plusWorkingDays() {
+        LocalDate start = LocalDate.of(2022, 9, 12);
+        when(nonWorkingDaysCollection.contains(start.plusDays(7))).thenReturn(true);
+        int days = 10;
+        Assertions.assertEquals(start.plusDays(15), calculator.plusWorkingDays(start, days));
```

Comment:
although i much prefer junit5, this class uses junit4 so it might be best to not mix the 2 in the same class and use the same assert the rest of the tests do